### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by abelagza**
In this milestone I:

• Copied the code from an old repository.
• Updated readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<label for="name" id="name-label">Enter your name:</label>
+<input type="text" id="name" placeholder="Enter a Name" required> <br> <br>
+
+
+
+<label for="email" id="email-label">Enter your email address:</label>
+<input type="email" id="email" placeholder="Enter an e-mail" required > <br><br>
+
+<label for="phone-number" id="number-label">Enter your phone-number:</label>
+<input type="number" id="number" placeholder="+1 222 222 2222" min="10" max="99" required><br> <br>
```
Comment (abelagza) [inline index.html]: Please refrain from using line breaks. `<br>`.
Instead, use the styling sheet to create spaces.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+<input type="submit" id="submit" value="Push Me!">
+
+</form>
+
+
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js">
+
+    </script>
```
Comment (abelagza) [inline index.html]: This repository is not supposed to have Scripts in it. 
Please remove, and stick to the instructions

***It was required, it is what checks that all requirements were met.***

**Review by abelagza — COMMENTED** [review]
# Status: Required changes
So close
### Highlights
✔️ No linter errors
✔️ Nice README
✔️ Good PR description
✔️ Good commit messages

### Required Changes ♻️
Check the comments under the review.

#### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by aimemalaika — APPROVED** [review]
Hi @abelagza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**